casper/src/test/scala/coop/rchain/casper/engine/RunningHandleBlockMessageSpec.scala

```diff
@@ -1,134 +0,0 @@
-package coop.rchain.casper.engine
```

Comment:
There should be BlockProcessor test instead of this... I guess I need to write one

Reply:
I suggest first to address concurrent block processing performance https://github.com/rchain/rchain/issues/3346 and then get back to writing tests

Reply:
https://github.com/rchain/rchain/issues/3346#issuecomment-818549965